protoc-gen-swagger/main.go

```diff
@@ -15,13 +15,15 @@ import (
 )
 
 var (
-	importPrefix          = flag.String("import_prefix", "", "prefix to be added to go package paths for imported proto files")
-	file                  = flag.String("file", "-", "where to load data from")
-	allowDeleteBody       = flag.Bool("allow_delete_body", false, "unless set, HTTP DELETE methods may not have a body")
-	grpcAPIConfiguration  = flag.String("grpc_api_configuration", "", "path to gRPC API Configuration in YAML format")
-	allowMerge            = flag.Bool("allow_merge", false, "if set, generation one swagger file out of multiple protos")
-	mergeFileName         = flag.String("merge_file_name", "apidocs", "target swagger file name prefix after merge")
-	useJSONNamesForFields = flag.Bool("json_names_for_fields", false, "if it sets Field.GetJsonName() will be used for generating swagger definitions, otherwise Field.GetName() will be used")
+	importPrefix               = flag.String("import_prefix", "", "prefix to be added to go package paths for imported proto files")
+	file                       = flag.String("file", "-", "where to load data from")
+	allowDeleteBody            = flag.Bool("allow_delete_body", false, "unless set, HTTP DELETE methods may not have a body")
+	grpcAPIConfiguration       = flag.String("grpc_api_configuration", "", "path to gRPC API Configuration in YAML format")
+	allowMerge                 = flag.Bool("allow_merge", false, "if set, generation one swagger file out of multiple protos")
+	mergeFileName              = flag.String("merge_file_name", "apidocs", "target swagger file name prefix after merge")
+	useJSONNamesForFields      = flag.Bool("json_names_for_fields", false, "if it sets Field.GetJsonName() will be used for generating swagger definitions, otherwise Field.GetName() will be used")
+	allowRepeatedFieldsInBody  = flag.Bool("allow_repeated_fields_in_body", false, "allows to use repeated field in `body` and `response_body` field of `google.api.http` annotation option")
```

Comment:
Does this parameter make sense in the swagger generator? I feel like many of these don't make sense in this file unless I misunderstand what it's for :thinking:. This just generates the `x.swagger.json`, right? Why should it be allowed to error if there's a repeated field in the body or a body on a DELETE method?

Reply:
(This isn't just directed at you, obviously, I can see the DELETE body flag was there already).

Reply:
Agreed, this was before the change `isPathParam` in [services.go](https://github.com/maros7/grpc-gateway/blob/repeated-field-path-parameter-support/protoc-gen-grpc-gateway/descriptor/services.go#L271). Will revert.